mlp/start_analysis.py

```diff
@@ -5,16 +5,25 @@
 from training.acceleration_dataset import AccelerationDataset
 from training.mlp_model import MLPMeasurementModel
 
+def writeModel2File(layers, output_filename):
```

Comment:
python style is lower case with underscores for function names and CamelCase for class names